### PR TITLE
Fix entire page capture 

### LINKED ISSUE #44: Capture entire page not working in chrome
Guys,

Please check the attached screenshot. Chrome version is "Version 61.0.3163.100 (Official Build) (64-bit)"

****
image

### PR DESCRIPTION
**Pull request by jkayani**
This PR fixes #44 and restores entire page capture functionality. 

This was broken because the extension wasn't properly causing the browser to scroll (`document.body.scroll*` instead of `document.documentElement.scroll*`), so that it could take captures all the way down the page.

The edit page was also incorrectly placing captures tiled along the x-axis b/c it was ignoring the device pixel ratio. 

In summary, I changed the `scrollNext()` function to properly scroll along the page, and some of the rendering code in `edit.js` to properly layout the captures into 1 image. 

Let me know if anything looks incorrect, or needs to be cleaned up.

### CODE REVIEW

**Comment by ralphcallaway** [comment]
tested with a couple simple pages and i'm now getting the full screen capture
